From b87124a7be6612112d62af1fead57de6303e2942 Mon Sep 17 00:00:00 2001
From: Luis Thomas <luis.thomas@silabs.com>
Date: Fri, 26 Aug 2022 15:06:41 +0200
Subject: [PATCH 3/3] experimental: added wsbrd to the --version message

added wsbrd to the --version message of dhcpd and dhcrelay,
this should allow us to discriminate between our patched version and
standard dhcpd/dhcrelay.
---
 relay/dhcrelay.c | 2 +-
 server/dhcpd.c   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/relay/dhcrelay.c b/relay/dhcrelay.c
index 946ef1b0..82858712 100644
--- a/relay/dhcrelay.c
+++ b/relay/dhcrelay.c
@@ -326,7 +326,7 @@ main(int argc, char **argv) {
 		if (!strcmp(argv[i], "-d")) {
 			no_daemon = 1;
 		} else if (!strcmp(argv[i], "--version")) {
-			log_info("isc-dhcrelay-%s", PACKAGE_VERSION);
+			log_info("isc-dhcrelay-wsbrd-%s", PACKAGE_VERSION);
 			exit(0);
 		} else if (!strcmp(argv[i], "--help") ||
 			   !strcmp(argv[i], "-h")) {
diff --git a/server/dhcpd.c b/server/dhcpd.c
index 3522889c..34f33046 100644
--- a/server/dhcpd.c
+++ b/server/dhcpd.c
@@ -329,7 +329,7 @@ main(int argc, char **argv) {
 			daemon = 0;
 #endif
 		} else if (!strcmp (argv [i], "--version")) {
-			const char vstring[] = "isc-dhcpd-";
+			const char vstring[] = "isc-dhcpd-wsbrd-";
 			IGNORE_RET(write(STDERR_FILENO, vstring,
 					 strlen(vstring)));
 			IGNORE_RET(write(STDERR_FILENO,
-- 
2.37.2

